src/Files.Uwp/Views/SettingsPages/Experimental.xaml

```diff
@@ -33,34 +33,7 @@
                 Padding="0,0,0,12"
                 FontStyle="Italic"
                 Text="{helpers:ResourceString Name=SettingsExperimentalDescription/Text}" />
-
-            <local:SettingsBlockControl
-                Title="{helpers:ResourceString Name=SettingsEnableFileTags/Title}"
-                HorizontalAlignment="Stretch">
-                <local:SettingsBlockControl.Icon>
-                    <FontIcon Glyph="&#xE1CB;" />
-                </local:SettingsBlockControl.Icon>
-                <ToggleSwitch IsOn="{Binding AreFileTagsEnabled, Mode=TwoWay}" Style="{StaticResource RightAlignedToggleSwitchStyle}" />
-                <local:SettingsBlockControl.ExpandableContent>
-                    <local:SettingsBlockControl
```

Comment:
We still need the option to edit tags, can you add it back?

Reply:
Ok no problem
Just one question: should I add this feature back in the settings or can I create a "Rename" command -that opens you the file- when you right click them in the sidebar?

Reply:
We want to add a proper UI that allows users to edit tags but in the meantime, we should just add the edit button back into settings. If you're interested in working on the edit UI, we should discuss it first and come up with a plan.

Reply:
Ok

Reply:
- I added the "Edit Tags" feature in the "Preference" section of "Settings"
- I removed faulty/not working options from the "Tags" on the Sidebar (Ex. "Open in another Window")
- I added this feature "allow the user to hide the "Tags" column from all folders at once" requested by @gave92 in a comment to #9586

Reply:
> I added the "Edit Tags" feature in the "Preference" section of "Settings"

This should still be under experimental for now 👍 

> I removed faulty/not working options from the "Tags" on the Sidebar (Ex. "Open in another Window")

We shouldn't remove them from the app, we should open an issue to track the bug.

Reply:
> I added this feature "allow the user to hide the "Tags" column from all folders at once" requested by @gave92 in a comment to

Thanks! Can you cherry pick this commit and open a separate PR for this?

Reply:
Thanks! Can you remove the commit from this branch?

Reply:
If you mean the "Hide Tags Column" I've already removed it from last commit